shaders/fragment.glsl

```diff
@@ -32,6 +32,7 @@ struct Fragment {
     vec4 color;
     vec4 gPosition;
     vec4 gNormal;
+    uint filterFlag;
```

Comment:
Seems that filtering was taken out of this branch -  I suppose this should go as well?